msal/src/main/java/com/microsoft/identity/nativeauth/UserAttributes.kt

```diff
@@ -29,17 +29,19 @@ import com.microsoft.identity.common.java.util.ObjectMapper
  * operation in Native Auth
  */
 class UserAttributes(internal val userAttributes: Map<String, String>) {
-    companion object Builder {
-        private const val CITY = "city"
-        private const val COUNTRY = "country"
-        private const val DISPLAY_NAME = "displayName"
-        private const val EMAIL_ADDRESS = "email"
-        private const val GIVEN_NAME = "givenName"
-        private const val JOB_TITLE = "jobTitle"
-        private const val POSTAL_CODE = "postalCode"
-        private const val STATE = "state"
-        private const val STREET_ADDRESS = "streetAddress"
-        private const val SURNAME = "surname"
+    class Builder {
+        companion object {
```

Comment:
Not an issue, just something to consider. Do we need the companion object here, what if simply make the constants members of the Builder class (declare them as val without the const keyword)?